abaplint.json

```diff
@@ -0,0 +1,330 @@
+{
+  "global": {
+    "files": "/src/**/*.*",
+    "exclude": ["demo_failures"],
+    "skipGeneratedGatewayClasses": true,
+    "skipGeneratedPersistentClasses": true,
+    "skipGeneratedFunctionGroups": true,
+    "useApackDependencies": false
+  },
+  "dependencies": [
+    {
+      "url": "https://github.com/abaplint/deps",
+      "folder": "/deps1",
+      "files": "/src/**/*.*"
+    },
+    {
+      "url": "https://github.com/abapGit/abapGit",
+      "folder": "/deps2",
+      "files": "/src/**/*.*"
+    }
+  ],
+  "syntax": {
+    "version": "v740sp08",
+    "errorNamespace": "^(Z|Y|LCL_|TY_|LIF_)",
```

Comment:
The project is under customer namespace (Y*). Can you please remove the `Y` from the `errorNamespace`?

Reply:
The pattern in errorNamespace is the objects that abaplint should report errors for

With this configuration, errors are issued if a YCL_ is not found, or a ZIF_ is not found, https://github.com/SAP/code-pal-for-abap/blob/8d6d5a8b80108ceee4b376a4b2e70078cdf673b5/src/foundation/y_code_pal_service.clas.abap#L85